source/adios2/toolkit/format/bp/BPBase.cpp

```diff
@@ -446,16 +446,15 @@ std::string BPBase::ReadBPString(const std::vector<char> &buffer,
 // static members
 const std::set<std::string> BPBase::m_TransformTypes = {
     {"unknown", "none", "identity", "bzip2", "sz", "zfp", "mgard", "png",
-     "blosc", "blosc2", "sirius", "mgardplus", "plugin"}};
+     "blosc", "sirius", "mgardplus", "plugin"}};
```

Comment:
Same question, I feel we need to be consistent with what we are keeping (if `blosc2` the naming should reflect this). Maybe I am misunderstanding this